test/test-gotest.sh

```diff
@@ -16,7 +16,13 @@ if [[ "$@" = *"--root"* ]]; then
 		exit 1
 	fi
 	XSUDO='sudo -E'
-	XTAGS='-tags root'
+	TAGS='root'
+fi
+
+# As per https://github.com/travis-ci/docs-travis-ci-com/blob/master/user/docker.md
+# Docker is not supported on Travis macOS test instances.
+if [[ "$TRAVIS_OS_NAME" == "osx" ]];then
+	TAGS="$XTAGS nodocker"
```

Comment:
Don't you mean $TAGS ? (Inside)

Reply:
Yes, but it is now refactored as an array.